[2023-12-14T08:38:36.964+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: restapi-pipeline.create_csv_external_stage manual__2023-12-14T08:38:29.673706+00:00 [queued]>
[2023-12-14T08:38:36.971+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: restapi-pipeline.create_csv_external_stage manual__2023-12-14T08:38:29.673706+00:00 [queued]>
[2023-12-14T08:38:36.971+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 3
[2023-12-14T08:38:36.984+0000] {taskinstance.py:1382} INFO - Executing <Task(SnowflakeOperator): create_csv_external_stage> on 2023-12-14 08:38:29.673706+00:00
[2023-12-14T08:38:36.991+0000] {standard_task_runner.py:57} INFO - Started process 259 to run task
[2023-12-14T08:38:37.002+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'restapi-pipeline', 'create_csv_external_stage', 'manual__2023-12-14T08:38:29.673706+00:00', '--job-id', '105', '--raw', '--subdir', 'DAGS_FOLDER/restapi.py', '--cfg-path', '/tmp/tmpbccgd0gg']
[2023-12-14T08:38:37.010+0000] {standard_task_runner.py:85} INFO - Job 105: Subtask create_csv_external_stage
[2023-12-14T08:38:37.056+0000] {task_command.py:416} INFO - Running <TaskInstance: restapi-pipeline.create_csv_external_stage manual__2023-12-14T08:38:29.673706+00:00 [running]> on host b5f695589db2
[2023-12-14T08:38:37.102+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sarthak' AIRFLOW_CTX_DAG_ID='restapi-pipeline' AIRFLOW_CTX_TASK_ID='create_csv_external_stage' AIRFLOW_CTX_EXECUTION_DATE='2023-12-14T08:38:29.673706+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-14T08:38:29.673706+00:00'
[2023-12-14T08:38:37.103+0000] {sql.py:274} INFO - Executing: 
CREATE OR REPLACE STAGE csv_s3_stage
URL='s3://my-random-bucket-002/coin-price-csv'
CREDENTIALS=(AWS_KEY_ID='AKIA4SXCAFHSOCMPU7LX' AWS_SECRET_KEY='DYlOTEHyogLcr0JylaJV8RdHRhCbUw9GFRzT/aEO')
DIRECTORY = (ENABLE = TRUE)
FILE_FORMAT = my_csv_format;
[2023-12-14T08:38:37.109+0000] {base.py:73} INFO - Using connection ID 'snowflake_default' for task execution.
[2023-12-14T08:38:37.115+0000] {base.py:73} INFO - Using connection ID 'snowflake_default' for task execution.
[2023-12-14T08:38:37.116+0000] {connection.py:334} INFO - Snowflake Connector for Python Version: 3.3.1, Python Version: 3.8.18, Platform: Linux-6.5.11-linuxkit-aarch64-with-glibc2.17
[2023-12-14T08:38:37.116+0000] {connection.py:1103} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2023-12-14T08:38:37.117+0000] {connection.py:1121} INFO - Setting use_openssl_only mode to False
[2023-12-14T08:38:37.547+0000] {cursor.py:833} INFO - query: [ALTER SESSION SET autocommit=False]
[2023-12-14T08:38:37.665+0000] {cursor.py:846} INFO - query execution done
[2023-12-14T08:38:37.666+0000] {cursor.py:988} INFO - Number of results in first chunk: 1
[2023-12-14T08:38:37.666+0000] {sql.py:418} INFO - Running statement: CREATE OR REPLACE STAGE csv_s3_stage
URL='s3://my-random-bucket-002/coin-price-csv'
CREDENTIALS=(AWS_KEY_ID='AKIA4SXCAFHSOCMPU7LX' AWS_SECRET_KEY='DYlOTEHyogLcr0JylaJV8RdHRhCbUw9GFRzT/aEO')
DIRECTORY = (ENABLE = TRUE)
FILE_FORMAT = my_csv_format;, parameters: None
[2023-12-14T08:38:37.667+0000] {cursor.py:833} INFO - query: [CREATE OR REPLACE STAGE csv_s3_stage URL='s3://my-random-bucket-002/coin-price-c...]
[2023-12-14T08:38:39.305+0000] {cursor.py:846} INFO - query execution done
[2023-12-14T08:38:39.306+0000] {cursor.py:988} INFO - Number of results in first chunk: 1
[2023-12-14T08:38:39.307+0000] {sql.py:427} INFO - Rows affected: 1
[2023-12-14T08:38:39.308+0000] {snowflake.py:400} INFO - Rows affected: 1
[2023-12-14T08:38:39.309+0000] {snowflake.py:401} INFO - Snowflake query id: 01b0faa6-3200-f6b5-0007-444200018076
[2023-12-14T08:38:39.309+0000] {cursor.py:833} INFO - query: [COMMIT]
[2023-12-14T08:38:39.404+0000] {cursor.py:846} INFO - query execution done
[2023-12-14T08:38:39.405+0000] {cursor.py:988} INFO - Number of results in first chunk: 1
[2023-12-14T08:38:39.405+0000] {connection.py:677} INFO - closed
[2023-12-14T08:38:39.473+0000] {connection.py:683} INFO - No async queries seem to be running, deleting session
[2023-12-14T08:38:39.601+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=restapi-pipeline, task_id=create_csv_external_stage, execution_date=20231214T083829, start_date=20231214T083836, end_date=20231214T083839
[2023-12-14T08:38:39.635+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-14T08:38:39.653+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
